resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesTestComponents.scala

```diff
@@ -33,7 +33,7 @@ import org.apache.spark.internal.config.JARS
 import org.apache.spark.internal.config.Tests.IS_TESTING
 import org.apache.spark.internal.config.UI.UI_ENABLED
 
-private[spark] class KubernetesTestComponents(defaultClient: DefaultKubernetesClient) {
+private[spark] class KubernetesTestComponents(val kubernetesClient: KubernetesClient) {
```

Comment:
Since there is no way to set `namespace` globally for the client in the new API, we can merge the `defaultClient` and `kubernetesClient`, the `val ` is reserved for `kubernetesClient` to minimize the change.

Before
```
class KubernetesTestComponents(defaultClient: DefaultKubernetesClient) {
  val kubernetesClient = defaultClient.inNamespace(namespace)
  ...
}
```
After
```
class KubernetesTestComponents(val kubernetesClient: KubernetesClient) {
  ...
}
```